frontend/src/css/GuestPage.styl

```diff
@@ -29,13 +29,16 @@
 .guestupload
   &__card
     &__form
+      display flex
+      flex-wrap wrap
+      width 100%
       &__left
-        width 50%
+        width 48%
```

Comment:
if you use flexbox, you are not supposed to use width percentage from 50% to 100% to allow a one line or two lines display depending on screen size.
you can keep flexbox mechanism but with proper usage